isObfuscated.py

```diff
@@ -0,0 +1,46 @@
+import fnmatch
+import os
+import glob
+from colorama import init
+import subprocess
+import os.path
+import random
+
+init()
+
+rootdir = './Extract/'
+
+
+class isObfuscated:
+	def obf_files(self):
+		count = 0
+		obfuscated = 0
+		not_obfuscated = 0
+		for subdir, dirs, files in os.walk(rootdir):
+			for file in files:
+				if(fnmatch.fnmatch(file, '*.java')):
+					count+=1
+					#print(file)
+					if (fnmatch.fnmatch(file, '[a-z].java') and file.islower()) or fnmatch.fnmatch(file,'[a-z][a-z]*.java') and file.islower():

```

Comment:
fnmatch.fnmatch(file, '[a-z].java') can be isolated in a helper methode called isFileNameLowCharacters